projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -242,6 +242,18 @@ function wpcom_launchpad_get_task_list_definitions() {
 				'share_site',
 			),
 		),
+		'site-setup'             => array(
```

Comment:
Might be worth calling this `legacy-site-setup` to make it clearer that it's an older/legacy task list.